app/src/main/java/br/com/concrete/tentacle/features/library/LibraryViewModel.kt

```diff
@@ -8,35 +8,40 @@ import br.com.concrete.tentacle.data.models.QueryParameters
 import br.com.concrete.tentacle.data.models.ViewStateModel
 import br.com.concrete.tentacle.data.models.library.Library
 import br.com.concrete.tentacle.data.repositories.LibraryRepository
+import br.com.concrete.tentacle.utils.Event
 
 class LibraryViewModel(private val libraryRepository: LibraryRepository) : BaseViewModel() {
 
-    private val viewStateLibrary: MutableLiveData<ViewStateModel<ArrayList<Library>>> = MutableLiveData()
+    private val viewStateLibrary: MutableLiveData<Event<ViewStateModel<ArrayList<Library>>>> = MutableLiveData()
     fun getLibrary() = viewStateLibrary
 
     @OnLifecycleEvent(Lifecycle.Event.ON_CREATE)
     fun preLoadLibrary() {
         loadLibrary()
     }
 
-    fun loadLibrary(queryParameters: QueryParameters? = null) {
+    fun loadLibrary(queryParameters: QueryParameters? = null, search: String? = null) {
         val query = queryParameters ?: QueryParameters()
```

Comment:
Esse método tá aceitando o queryParameters como null, mas em seguida se for null ele cria uma instância do objeto. Realmente faz sentido aceitar null?